kola/tests/misc/tang.go

```diff
@@ -29,14 +29,13 @@ const (
 			"version": "3.3.0"
 		},
 		"kernelArguments": {
-			"shouldExist": ["rd.luks.name=12345678-9abc-def0-1234-56789abcdef0=rootencrypted", "rd.networkd=1", "systemd.mask=systemd-cryptsetup@rootencrypted.service"]
+			"shouldExist": ["rd.networkd=1", "systemd.mask=systemd-cryptsetup@rootencrypted.service"]
```

Comment:
```suggestion
			"shouldExist": ["rd.networkd=1"]
```
The masking can also go, or? I think it didn't work and if it would then unlocking wouldn't work.

Reply:
You are right, the masking can go. I removed it.

Aside:

> if it would then unlocking wouldn't work

`systemd.mask` only masks units during main boot, as opposed to `rd.systemd.mask`, which masks units in initramfs ([docs](https://www.freedesktop.org/software/systemd/man/latest/systemd-debug-generator.html#Description)). Thus, the `systemd-cryptsetup@rootencrypted` in initramfs wouldn't be masked by this cmdline parameter, and unlocking in initramfs would work. I thought I had seen that systemd creates a second `systemd-cryptsetup@rootencrypted` for the main boot, too, because Ignition added an entry for the root partition to `/etc/crypttab`, and that this second unit failed because the root partition was already unlocked during the initramfs. But I tested it and apparently, that's not the case. So, the masking is superfluous, as you suggested.

Reply:
Thanks for checking this, glad we can get rid of it :)